### PR TITLE
Add missing env var name in hypershift powervs release prow job

### PR DESCRIPTION
**Pull request by dharaneeshvrd**

### CODE REVIEW

**Review by hoxhaeris — APPROVED** [review]

**Review by mkumatag — APPROVED** [review]
/lgtm

File: ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml
Comment (hoxhaeris) [inline ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1]: This should be changed to amd64.

Comment (hoxhaeris) [inline ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml:1]: This should be changed to amd64.

**Pull request merged**